diff --git a/homeassistant/auth/__init__.py b/homeassistant/auth/__init__.py
index 3ed24056b5..867066e4fb 100644
--- a/homeassistant/auth/__init__.py
+++ b/homeassistant/auth/__init__.py
@@ -673,7 +673,7 @@ class AuthManager:
             jwt_wrapper.verify_and_decode(
                 token, jwt_key, leeway=10, issuer=issuer, algorithms=["HS256"]
             )
-        except jwt.InvalidTokenError:
+        except (jwt.InvalidTokenError, jwt.InvalidKeyError):
             return None
 
         if refresh_token is None or not refresh_token.user.is_active:
